src/components/editPostModal/index.jsx

```diff
@@ -21,22 +23,38 @@ const EditPostModal = () => {
         }, 2000)
     }
 
+    const userInitials =
```

Comment:
We're repeating ourselves here. Consider splitting this into a component. In either case, still remove the amount of `||`